resources/testdriver.js

```diff
@@ -1066,6 +1066,27 @@
          */
         clear_device_posture: function(context=null) {
             return window.test_driver_internal.clear_device_posture(context);
+        },
+
+        /**
+         * Run the deletion algorithm for all hosts in the stateful bounce
+         * tracking map (without regard for the bounce tracking grace period),
+         * then return a list of those hosts.
+         *
+         * Matches the `Run Bounce Tracking Mitigations
+         * https://privacycg.github.io/nav-tracking-mitigations/#bounce-tracking-mitigations`_
+         * WebDriver command.
+         *
+         * @param {WindowProxy} [context=null] - Browsing context in which to
+         *                                       run the call, or null for the
+         *                                       current browsing context.
+         * @returns {Promise} Fulfilled after the deletion algorithm has
```

Comment:
Please also change this case of "deletion algorithm" if the description above is updated.